dev/breeze/doc/01_installation.rst

```diff
@@ -234,8 +234,19 @@ In case of disk space errors on macOS, increase the disk space available for Doc
 
 Installation
 ============
+First, clone the Airflow repository, but ensure you do not clone it into your home directory. Doing so will trigger the following error:
+'Your Airflow sources are checked out in /Users/username/airflow, which is also your AIRFLOW_HOME where Airflow writes logs and database files. This setup is problematic because Airflow might overwrite or clean up your source code and .git repository.'
```

Comment:
```suggestion
`Your Airflow sources are checked out in /Users/username/airflow, which is also your AIRFLOW_HOME where Airflow writes logs and database files. This setup is problematic because Airflow might overwrite or clean up your source code and .git repository.`
```